server/config/raw/repo_cfg.go

```diff
@@ -41,12 +41,12 @@ func (r RepoCfg) Validate() error {
 	return validation.ValidateStruct(&r,
 		validation.Field(&r.Version, validation.By(equals2)),
 		validation.Field(&r.Projects),
-		validation.Field(&r.WorkflowModeType, validation.In("pr", "platform")),
+		validation.Field(&r.WorkflowModeType, validation.In("platform")),
 	)
 }
 
 func (r RepoCfg) ToValid() valid.RepoCfg {
-	workflowModeType := toWorkflowModeType(r.WorkflowModeType)
+	workflowModeType := toWorkflowModeType()
```

Comment:
shall we just delete this method?  seems like it isn't doing any proper conversion anymore, it's just returning a static value